__tests__/app.test.js

```diff
@@ -242,6 +242,47 @@ describe("/api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe("/api/reviews/:review_id/comments", () => {
+  test("201: responds with the posted comment", () => {
+    const post = {
+      username: "mallionaire",
+      body: "Percy loves playing chess.",
+    };
+    return request(app)
+      .post("/api/reviews/2/comments")
+      .send(post)
+      .expect(201)
+      .then((response) => {
+        const comment = response.body.comment[0];
+        expect(comment.hasOwnProperty("comment_id")).toBe(true);
+        expect(comment.hasOwnProperty("body")).toBe(true);
+        expect(comment.hasOwnProperty("review_id")).toBe(true);
+        expect(comment.hasOwnProperty("author")).toBe(true);
+        expect(comment.hasOwnProperty("votes")).toBe(true);
+        expect(comment.hasOwnProperty("created_at")).toBe(true);
+        expect(comment.review_id).toBe(2);
+        expect(comment.author).toBe("mallionaire");
+        expect(comment.body).toBe("Percy loves playing chess.");
+        expect(comment.comment_id).toBe(7);
```

Comment:
This checks all the key:value pairs on the object, but you could make it more succinct by doing something like:
expect(comment).toEqual( {comment_id: 123, body: "lalala"....})